core/templates/dev/head/components/forms/html_select_directive.html

```diff
@@ -1,12 +1,11 @@
 <div uib-dropdown class="oppia-html-select">
   <button uib-dropdown-toggle class="btn btn-default oppia-html-select-selection" type="button" aria-expanded="false" data-toggle="dropdown">
     <div>
-      <angular-html-bind class="oppia-html-select-selection-element" html-data="options[getSelectionIndex()].val">
-      </angular-html-bind>
+      Click here to select
```

Comment:
Why are you changing the existing view? I mean in your case the author would see the `Click here to select` whereas in original he would see the first option. 

I feel, we should keep the original view intact and probably increase the size of the box (where you currently right click here to select) according to the length of first option. The bigger the option the bigger the box gets (using word-break etc.). We can discuss upon this one.

Reply:
We can rather change the existing view to a view that initially shows the first option, but truncated with 3 dots at the end. In as many websites I have seen till date, none of them increase the size of the drop-down select box. It doesn't look clean to me. What do you think?
Thanks!

Reply:
@seanlip Would also like to take your suggestion here.

Reply:
I think it would be better to have it truncated in the display. Having a horizontally scrollable dropdown looks weird. (Correct me if I'm wrong, but from the screenshots it looks like it is horizontally scrollable right?). Also, I don't think this is very likely to happen in production as it is not normal to have such long words. 

Or is it possible to add something similar to word-break to get the same functionality as the rtes? I am happy either way, but yeah, I would prefer the word-break approach so that it looks similar on all views.

Reply:
@nithusha21 Alright. Working on the changes.

Reply:
![image](https://user-images.githubusercontent.com/32777813/50055146-bc61c600-0170-11e9-884b-7209ca0093d6.png)
@nithusha21 @bansalnitish 
Does this look fine now?

Reply:
@nithusha21 size of dropdown is matched now. Have a look. Thanks.
![image](https://user-images.githubusercontent.com/32777813/50144103-5726e580-02d4-11e9-93aa-334154b67009.png)

Reply:
Looks better now! Thanks :)